readthedocs/projects/views/public.py

```diff
@@ -220,6 +222,15 @@ def project_download_media(request, project_slug, type_, version_slug):
         slug=version_slug,
     )
 
+    # Send media download to analytics - sensitive data is anonymized
+    analytics_event.delay(
+        event_category='Build Media',
+        event_action=f'Download {type_}',
+        event_label=str(version),
+        ua=request.META.get('HTTP_USER_AGENT'),
+        uip=get_client_ip(request),
```

Comment:
Do we want to send this?

Reply:
If we don't care about it, we could definitely drop it. It is anonymized but it also isn't really useful. I'd be fine dropping it.

Reply:
Up to you. I'm 👍 on dropping it if we don't need it though.

Reply:
I'll update. Let's drop it. Seems cleaner that way especially since we don't need it. I'll just verify that GA doesn't silently drop requests without them.

Reply:
In my testing, without sending *something* for the user agent and IP address, the event doesn't show up in the realtime reporting. I'll give it a day to see if it does show up in the aggregated reporting but we may need to send some value for those settings. That could be the server's IP address.

Reply:
Alright, let's just ship it and let GA anonymize it, we're already doing it elsewhere.

Reply:
After looking at GA's regular reporting, only the events with an IP and UA were counted.